CDP4PluginPackager/CDP4PluginPackager.csproj

```diff
@@ -21,7 +21,7 @@
 	</PropertyGroup>
 	<Target Name="PostBuild" AfterTargets="PostBuildEvent" />
 	<ItemGroup>
-		<PackageReference Include="CDP4Common-CE" Version="27.4.1-rc1" />
+		<PackageReference Include="CDP4Common-CE" Version="28.0.0-rc26" />
 		<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
do we still need newtonsoft here?

Reply:
Refactored this, so not needed anymore. Only SDK is refactored to System.Text.Json, not all IME code, so IME still uses NewtonSoft in some parts.